src/main/java/frc/team2412/robot/Drivetrain.java

```diff
@@ -27,12 +27,10 @@
 
   private final AnalogGyro m_gyro = new AnalogGyro(0);
 
-  private final SwerveDriveKinematics m_kinematics =
-      new SwerveDriveKinematics(
-          m_frontLeftLocation, m_frontRightLocation, m_backLeftLocation, m_backRightLocation);
+  private final SwerveDriveKinematics m_kinematics = new SwerveDriveKinematics(
+      m_frontLeftLocation, m_frontRightLocation, m_backLeftLocation, m_backRightLocation);
 
-  private final SwerveDriveOdometry m_odometry =
-      new SwerveDriveOdometry(m_kinematics, m_gyro.getRotation2d());
+  private final SwerveDriveOdometry m_odometry = new SwerveDriveOdometry(m_kinematics, m_gyro.getRotation2d());
 
```

Comment:
Check in [the formatter settings](https://github.com/redhat-developer/vscode-java/wiki/Formatter-settings) you used so we can update it as needed & use the same formatting throughout.

Reply:
I haven't changed formatter settings, so I assume they're default

Reply:
Ran "Java: Open Java Formatter Settings" in VSCode and added the format settings file & setting config option it added.